### PR TITLE
fix(quantic): Fix flaky test in E2E facet tests

### PR DESCRIPTION
**Pull request by aelfaleh**
image
image
image
image

### CODE REVIEW

File: packages/quantic/cypress/integration/facets/facet/facet.cypress.ts [hunk 1]
```diff
@@ -381,11 +383,14 @@ describe('Facet Test Suite', () => {
...
-        label: defaultLabel,
-        displayValuesAs: 'link',
-      });
+      visitFacetPage(
+        {
+          field: defaultField,
+          label: defaultLabel,
+          displayValuesAs: 'link',
+        },
+        false
```
Comment (nathanlb) [inline packages/quantic/cypress/integration/facets/facet/facet.cypress.ts:392]: Could you help me understand why we explicitely don't want to wait for search to return in this case?

Comment (aelfaleh) [reply packages/quantic/cypress/integration/facets/facet/facet.cypress.ts:392]: I have expectations that need to wait for search

Comment (nathanlb) [reply packages/quantic/cypress/integration/facets/facet/facet.cypress.ts:392]: The first assertion in the test seems to assume the search has returned though. Could you show me which one requires it please?
```
Expect.displayPlaceholder(false);
```

File: packages/quantic/cypress/integration/facets/facet/facet.cypress.ts [hunk 2]
```diff
@@ -423,6 +431,7 @@ describe('Facet Test Suite', () => {
 
         function collapseFacet() {
           Actions.clickCollapseButton();
+          cy.wait(InterceptAliases.Search);
```
Comment (lbergeron) [inline packages/quantic/cypress/integration/facets/facet/facet.cypress.ts:434]: Collapsing a facet does not trigger a search query. We shouldn't wait here.

**Review by lbergeron — APPROVED** [review]
Well done!

**Review by lvu285 — APPROVED** [review]

File: packages/quantic/cypress/integration/facets/facet/facet.cypress.ts (hunk 1 above)
Comment (aelfaleh) [reply packages/quantic/cypress/integration/facets/facet/facet.cypress.ts:392]: `    aliasFacetValues();
        Expect.facetValuesEqual(indexFacetValuesAlias);`

` function aliasFacetValues() {
    cy.wait(InterceptAliases.Search).then((interception) => {
      const indexValues = extractFacetValues(interception.response);
      cy.wrap(indexValues).as(indexFacetValuesAlias.substring(1));
    });
  }`

**Pull request merged**